TestProjects/UniversalGraphicsTest_Foundation/Assets/Scenes/054_Lighting_Attenuation/Lightmap-0_comp_light.exr.meta

```diff
@@ -0,0 +1,96 @@
+fileFormatVersion: 2
```

Comment:
Are the png, meta, and asset file changes coming with the backports?

Reply:
Thanks for catching that!